### PR TITLE
Refactor and remove unused components

### PR DESCRIPTION
**Pull request by Diogomartf**
# Summary

- Removes unused components from our codebase. 
- Renames components.
- Moves add and remove liquidity to pools folder.

  # To Test

1. No visible changes. Everything should work as expected.

### CODE REVIEW

**Comment by niemam29** [comment]
Tested and looks good to me

**Review by mix1o — APPROVED** [review]

File: src/components/Pool/PoolLiquidityHeader/PoolLiquidityHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { RowBetween } from '../../Row'
+import { Settings } from '../../Settings'
+
+export function PoolLiquidityHeader({ adding, creating }: { adding: boolean; creating: boolean }) {
+  const { navigate } = useRouter()
+  return (
+    <Tabs>
+      <RowBetween mb="16px">
+        <StyledArrowLeft onClick={() => navigate(-1)} />
+        <ActiveText>{creating ? 'Create a pair' : adding ? 'Add Liquidity' : 'Remove Liquidity'}</ActiveText>
```
Comment (jorgedclopes) [inline src/components/Pool/PoolLiquidityHeader/PoolLiquidityHeader.tsx:14]: Use translations.

**Review by jorgedclopes — APPROVED** [review]
LGTM

**Review by berteotti — APPROVED** [review]
Looks good!

File: src/components/Pool/PoolLiquidityHeader/PoolLiquidityHeader.tsx (hunk 1 above)
Comment (Diogomartf) [reply src/components/Pool/PoolLiquidityHeader/PoolLiquidityHeader.tsx:14]: Fixed!

**Pull request merged**